src/components/Gallery/Gallery.tsx

```diff
@@ -73,16 +81,12 @@ const Gallery = ({ images }) => {
         images={formattedArray}
         index={index}
         modalIsOpen={modalOpen}
-        toggleModal={toggleModal}
+        toggleModal={() => toggleModal}
```

Comment:
why this change?

Reply:
due to TS warning of toggleModal being the wrong type.
Type '(selectedIndex: number) => void' is not assignable to type '(event: SyntheticEvent<HTMLButtonElement, Event>) => void'.